### ISSUE TITLE
No Response

### ISSUE THREAD

**Issue by mistak1992** [hannseman/homebridge-mi-hygrothermograph#90]
No Response

### Current Behavior
No Response

### Steps to Reproduce (for bugs)

run homebridge

### Versions

- **homebridge**: 0.4.50
- **homebridge-mi-hygrothermograph**: homebridge-mi-hygrothermograph@2.3.2
- **node**:       v12.14.0
- **OS**:         Ubuntu 18.04

### Configuration

```

{
  "bridge":{
    "name":"Homebridge",
    "username":"CD:22:3D:E3:CE:30",
    "port":51826,
    "pin":"031-45-154"
  },
  "description":"Homebridge",
  "accessories":[
        {
       "accessory": "Thermostat",
       "name": "Gree",
       "apiroute": "http://127.0.0.1:8086",
       "maxTemp": 30,
       "minTemp": 17,
       "model": "KFR-26G",
       "serial": "201912180014004",
       "manufacturer": "Gree"
     },{
       "accessory": "Hygrotermograph",
       "name": "Temperature & Humidity"
     }
  ],
  "platforms":[
  ]
}


```

### Homebridge Log / Command Output
```
[12/31/2019, 6:37:47 AM] [Temperature & Humidity] Error: Unknown event type null
    at Scanner.onDiscover (/usr/local/lib/node_modules/homebridge-mi-hygrothermograph/lib/scanner.js:134:28)
    at Noble.emit (events.js:210:5)
    at Noble.onDiscover (/usr/local/lib/node_modules/homebridge-mi-hygrothermograph/node_modules/@abandonware/noble/lib/noble.js:178:10)
    at NobleBindings.emit (events.js:210:5)
    at NobleBindings.onDiscover (/usr/local/lib/node_modules/homebridge-mi-hygrothermograph/node_modules/@abandonware/noble/lib/hci-socket/bindings.js:174:10)
    at Gap.emit (events.js:210:5)
    at Gap.onHciLeAdvertisingReport (/usr/local/lib/node_modules/homebridge-mi-hygrothermograph/node_modules/@abandonware/noble/lib/hci-socket/gap.js:256:10)
    at Hci.emit (events.js:210:5)
    at Hci.processLeAdvertisingReport (/usr/local/lib/node_modules/homebridge-mi-hygrothermograph/node_modules/@abandonware/noble/lib/hci-socket/hci.js:651:10)
    at Hci.processLeMetaEvent (/usr/local/lib/node_modules/homebridge-mi-hygrothermograph/node_modules/@abandonware/noble/lib/hci-socket/hci.js:608:10)
    at Hci.onSocketData (/usr/local/lib/node_modules/homebridge-mi-hygrothermograph/node_modules/@abandonware/noble/lib/hci-socket/hci.js:479:12)
    at BluetoothHciSocket.emit (events.js:210:5)

```

**Comment by fluidof** [comment]
same to me

**Comment by andreyxc** [comment]
same to me

**Comment by hannseman** [comment]
@mistak1992 is this ubuntu running on a Raspberry pi? Did this plugin work before and stopped working after updating your system?

@fluidof are you too running ubuntu? 

Could you please provide more detailed logs by running homebridge in debug mode (`use the -D flag`).

**Comment by fluidof** [comment]
> @fluidof are you too running ubuntu?
> 
> Could you please provide more detailed logs by running homebridge in debug mode (use the `-D` flag).

I'm on Raspberry Pi

**Comment by hannseman** [comment]
I just released version `2.4.0` to npm. Please try and install this version instead as it has support for node 12.

**Comment by fluidof** [comment]
> I just released version `2.4.0` to npm. Please try and install this version instead as it has support for node 12.

Same problem but I have installed nodes 13.5

**Comment by martano12** [comment]
I have no data since installed on HOOBS 3.1.1 until now, 3.1.19 on a RPI 3B+

**Comment by hannseman** [comment]
It would help if I had some debug logs so I could actually see the transmitted data that could contain the event. Could you run homebridge with the debug-flag `homebridge -D`

**Comment by RAZRu** [comment]
same, No Response

**Comment by mistak1992** [comment]
unfortunately, my ubuntu server have been removed from vbox. but i redeploy it on my NAS server.(OMV, debain 9) 
nodejs 12.15.0
homebridge-mi-hygrothermograph@2.4.0 
still No Response
but no more error msg.

**Comment by dgproductions88** [comment]
hi there, thanks for the plugin. I've just installed but not working.... i'm running homebridge with raspbian and here my config:
node.js - v12.14.1
npm - v.6.13.6

I'm facing the next:

1: With no ble enabled in the hygrotherm:

`feb 28 00:02:09 raspberrypi homebridge[16103]: [2020-2-28 0:02:09] [Config] [homebridge-mi-hygrothermograph] Error: ESOCKETTIMEDOUT
`

2: if i enable the bluetooth for discover it:

`
feb 28 00:04:05 raspberrypi homebridge[16103]: [2020-2-28 0:04:05] [Temperature & Humidity] Error: Unknown event type: 2. 7122aa015cb19838342d580d0200020d10
feb 28 00:04:05 raspberrypi homebridge[16103]:     at Parser.parseEventData (/usr/local/lib/node_modules/homebridge-mi-hygrothermograph/lib/parser.js:174:15)
feb 28 00:04:05 raspberrypi homebridge[16103]:     at Parser.parse (/usr/local/lib/node_modules/homebridge-mi-hygrothermograph/lib/parser.js:56:23)
feb 28 00:04:05 raspberrypi homebridge[16103]:     at Scanner.parseServiceData (/usr/local/lib/node_modules/homebridge-mi-hygrothermograph/lib/scanner.js:164:38)
feb 28 00:04:05 raspberrypi homebridge[16103]:     at Scanner.onDiscover (/usr/local/lib/node_modules/homebridge-mi-hygrothermograph/lib/scanner.js:90:25)
feb 28 00:04:05 raspberrypi homebridge[16103]:     at Noble.emit (events.js:223:5)
feb 28 00:04:05 raspberrypi homebridge[16103]:     at Noble.onDiscover (/usr/local/lib/node_modules/homebridge-mi-hygrothermograph/node_modules/@abandonware/noble/lib/noble.js:178:10)
feb 28 00:04:05 raspberrypi homebridge[16103]:     at NobleBindings.emit (events.js:223:5)
feb 28 00:04:05 raspberrypi homebridge[16103]:     at NobleBindings.onDiscover (/usr/local/lib/node_modules/homebridge-mi-hygrothermograph/node_modules/@abandonware/noble/lib/hci-socket/bindings.js:174:10)
feb 28 00:04:05 raspberrypi homebridge[16103]:     at Gap.emit (events.js:223:5)
feb 28 00:04:05 raspberrypi homebridge[16103]:     at Gap.onHciLeAdvertisingReport (/usr/local/lib/node_modules/homebridge-mi-hygrothermograph/node_modules/@abandonware/noble/lib/hci-socket/gap.js:256:10)
`

I've installed all the dependencies but still not working.

I think that it's related to this thread but... if you think that needs an isolated one.... please tell me.

Thanks in advance!

**Comment by hannseman** [comment]
I just released a new version (3.0.1) with updates to the bluetooth library. Please give it a shot.

**Comment by hannseman** [comment]
Closing this issue due to no response.
